client/devdocs/controller.js

```diff
@@ -137,7 +135,7 @@ const devdocs = {
 	},
 
 	pleaseLogIn: function ( context, next ) {
-		const currentUrl = url.parse( window.location.href );
+		const currentUrl = new URL( window.location.href );
```

Comment:
No need to turn this into a native URL if you don't want to. `window.location` already has all the fields that are accessed below, e.g. `window.location.protocol`. This is fine too, though.

Reply:
Yeah, I wasn't sure why it's being done so I decided to keep the same logic and just replace the means to get there 😉